src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs

```diff
@@ -344,13 +346,13 @@ async void UpdateRepositoryList(object sender, NotifyCollectionChangedEventArgs
                         .Cast<ILocalRepositoryModel>()
                         .ForEach(async r =>
                     {
-                        if (Equals(Holder.ActiveRepo, r))
+                        if (Equals(holder.TeamExplorerContext.ActiveRepository, r))
```

Comment:
Is this comparison still valid? Previously the repository models overrode `Equals`, but now they don't. I think we need to decide what "equals" means in this case? Same `CloneUri` and `LocalPath`?

Reply:
Sorry, the comparison logic is still in there for now. Ignore this.